qcodes_contrib_drivers/drivers/Agilent/Agilent_N9000A.py

```diff
@@ -0,0 +1,109 @@
+# This Python file uses the following encoding: utf-8
+# Etienne Dumur <etienne.dumur@gmail.com>, august 2020
+
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import Numbers
+
+
+class Agilent_N9000A(VisaInstrument):
+    """
+    This is the QCoDeS python driver for the Agilent CXA N9000A power spectrum
+    analyzer.
+    """
+    
+    
+    
+    def __init__(self, name       : str,
```

Comment:
You have a lot of blank lines here. Most python style guids would recommend only one or two blank lines depending on the situation

Reply:
Normally corrected, please tell me if that format suits you better.